tests/test_operator.py

```diff
@@ -588,10 +584,25 @@ def test_argument_derivation_order(self, nt=100):
 
         # Providing a scalar argument explicitly should override the automatically\
         # inferred
-        op_arguments = op.arguments(b=b1, time=nt - 1, time_e=nt - 2)
+        op_arguments = op.arguments(b=b1, time_e=nt - 2)
```

Comment:
Why has this been changed?

Reply:
So Michael changed it; I **think** it's because now ``time`` aliases to ``time_e``